examples/base/game.py

```diff
@@ -53,16 +53,24 @@
 for name, config in agent_vis_config.items():
     ctx.visual.set_agent_visual(name, **config)
 
-
-
-
-# turn_count = 0
-# # Rules for the game
-# def rule_terminate(ctx):
-#     if turn_count > 100:
-#         ctx.terminate()
-#         return True
-#     return False
+# Special nodes
+n1 = ctx.graph.graph.get_node(0)
+n2 = ctx.graph.graph.get_node(1)
+data = {}
+data['x'] = n1.x
+data['y'] = n1.y
+data['scale'] = 10.0
+data['color'] = (0.0, 0.0, 0.0, 0.3)
```

Comment:
For demo purpose, can we make this color visible? For example `(255, 0, 0)`
```suggestion
data['color'] = (255, 0, 0)
```